terms.adoc

```diff
@@ -0,0 +1,130 @@
+////
+Apply pseudo legal capitalization to the main document:
+cat eclipse_development_process.adoc | sed -E -e "`grep -P -oh "^.+(?=\s+\:\:)" terms.adoc | while read -r term; do echo -ne "s|$term|$term|gI;"; done`"
+////
+
+[glossary]
+
+Architecture Council ::
+The Eclipse Architecture Council (AC) is a council of experienced Eclipse committers responsible for identifying and tackling any issues that hinder continued technological success and innovation, widespread adoption, and future growth of Eclipse technology.
+
+Board of Directors ::
+The business and technical affairs of the Eclipse Foundation are managed by or under the direction of the Eclipse Board of Directors. 
+
+Check Point Review ::
+Check Point Review is a category of Review that includes: Creation Review, Progress Review, Release Review, and Graduation Review.
+
+Committer ::
+A committer is a software developer who has the necessary rights to make decisions regarding a Project.
+
+Contributor ::
+A Contributor is anybody who makes contributions to the project.
+
+Creation Review ::
+A Creation Review is a type of Review that marks a Project's transition from the Proposal Phase to the Incubation Phase.
+
+Developer ::
+A Committer or Contributor.
+
+Ecosystem ::
+A commercial Ecosystem is a system in which companies, organizations, and individuals all work together for mutual benefit.
+
+Eclipse Management Organization ::
+The Eclipse Management Organization (EMO) consists of the Eclipse Foundation staff, and the Eclipse Architecture and Planning Councils. 
+
+Graduation Review ::
+A Graduation Review is a type of Review that marks a Project's transition from the Incubation Phase to the Mature Phase.
+
+Incubation Branding ::
+Incubation Branding is the branding that must be applied to all projects in the Incubation Phase.
+
+Incubation Phase ::
+The Incubation Phase is a Phase during which the Project Team develops the process, the community, and the technology for a new Project.
+
+Integration Build ::
+An Integration Build is a build of the project content that is generated periodically (generally weekly) and made available to the Developer community for testing purposes.
```

Comment:
The schedule based cadence here is usually wrong. Many projects do schedule daily I-Builds, many other projects do provide snapshot builds for integration whenever there is a commit. I think we should omit the "periodically" and "weekly" part.

Reply:
+1